alpha/dotnet-chart/values.yaml

```diff
@@ -23,7 +23,7 @@ build:
   ## The ImageStreamTag of the desired builder image
   imageStreamTag:
     ## Name of the ImageStreamTag
-    name: dotnet:3.1
+    name: dotnet:8.0
```

Comment:
The 3.1 version that was still used in this chart went EOL December 2022.
The 8.0 version that is now used is supported for the next 3 years.